modules/Bio/Vega/Transform/RegionToXML.pm

```diff
@@ -277,6 +286,27 @@ sub generate_Transcript {
         $t->attribvals($self->prettyprint('stable_id',$tran->stable_id));
     }
 
+    # Fetch transcript attributes that do not persist
+    my %hash;
+    my @current_persistent_attributes = ['name', 'remark', 'hidden_remark', 'cds_start_NF', 'cds_end_NF', 'mRNA_start_NF', 'mRNA_end_NF', 'status'];
```

Comment:
It seems to me that you are storing a reference to an array into an array, I don't see any reason/advantage